shadow.c

```diff
@@ -117,10 +118,19 @@ static PHP_GSHUTDOWN_FUNCTION(shadow)
 }
 /* }}} */
 
+/* {{{ shadow_deps
+ */
+static const zend_module_dep shadow_deps[] = {
+	ZEND_MOD_OPTIONAL("zip")
```

Comment:
It's not exactly `Makefile` that breaks our test. Here's a piece of code that puts `zip` extension after `shadow`: https://github.com/php/php-src/blob/PHP-5.4.45/run-tests.php#L1472

We could patch `run-tests.php` on the fly, but it's rather hacky and unreliable.